invenio_opendata/base/recordext/fields/opendata.cfg

```diff
@@ -305,6 +321,14 @@ reprocessing_date:
     producer:
         json_for_marc(), {"960__c": ""}
 
+# 964_0c
+
+run_number:
```

Comment:
A "run number" is usually an integer such as 165467.  Here we are storing "run period" values, such as `Run2011A`. @katilp may advise on the best term to use...